spot_driver/launch/spot_driver.launch.py

```diff
@@ -10,13 +10,15 @@
 from launch.launch_description_sources import PythonLaunchDescriptionSource
 from launch.substitutions import Command, FindExecutable, LaunchConfiguration, PathJoinSubstitution, TextSubstitution
 from launch_ros.substitutions import FindPackageShare
+from synchros2.launch.actions import DeclareBooleanLaunchArgument, update_sigterm_sigkill_timeout
 
 from spot_driver.launch.spot_launch_helpers import IMAGE_PUBLISHER_ARGS, declare_image_publisher_args, spot_has_arm
 
 THIS_PACKAGE = "spot_driver"
 
 
 def launch_setup(context: LaunchContext, ld: LaunchDescription) -> None:
+    update_sigterm_sigkill_timeout(ld)
```

Comment:
question -- how does this change how the driver shuts down on robot?

Reply:
It just increases the allotted time for shutting down. I didn't notice any side effects while testing but maybe?

Reply:
ah cool I was worried that if it decreased the allotted time for shutting down that some stuff might not shut down cleanly (esp in the ros2 control case)